### ISSUE TITLE
Accessibility Report Summary does not show total errors.

### ISSUE THREAD

**Issue by R3dXIII** [pa11y/pa11y-ci#169]
I don't know if it's ok to post issues about pa11y-ci-reporter-html, but I couldn't find any special repo for it.

The Accessibility Report Summary is nice, but it lacks the total which is at the top of the xml file generated by pa11y-ci:

"total": 14,
"passes": 0,
"errors": 434,

**Comment by aarongoldenthal** [comment]
@R3dXIII The pa11y-ci-reporter-html repository is at https://gitlab.com/gitlab-ci-utils/pa11y-ci-reporter-html, you can also find the link on the [npm package page](https://www.npmjs.com/package/pa11y-ci-reporter-html). I'm always glad to take issues there, but I created one for this at https://gitlab.com/gitlab-ci-utils/pa11y-ci-reporter-html/-/issues/73.

**Comment by R3dXIII** [comment]
Thanks @aarongoldenthal , hopefully the summary will get added now.

**Comment by josebolos** [comment]
Closing issue. Please follow https://gitlab.com/gitlab-ci-utils/pa11y-ci-reporter-html/-/issues/73 for updates!
